Samples~/SampleGame/Assets/Scripts/Server/GameLiftServer.cs

```diff
@@ -12,6 +18,9 @@ public class GameLiftServer
 {
     private readonly GameLift _gl;
     private readonly Logger _logger;
+    private readonly Settings _settings;
+    private readonly ICredentialsStore _credentialsStore;
```

Comment:
Wrap all usage of the credentials store with `#if UNITY_EDITOR` since it's only used for that auth token shortcut